docs/docusaurus/versioned_docs/version-1.3.0/orc8r/upgrade_1_3.md

```diff
@@ -77,6 +77,8 @@ upgraded and Prometheus should be back up on v2.20.1.
 
 ## Data Migrations
 
+> **_NOTE:_** If you're upgrading to release v1.3.0 specifically, `m014_enodeb_config` is not required.
```

Comment:
Can you make it "... upgrading to release tag v1.3.0 specifically" -- just to clarify this only applies to the tag (not the v1.3 patch branch or subsequent tags)